staff/templates/coupon/edit.html

```diff
@@ -0,0 +1,32 @@
+{% extends './base.html' %}
+{% load crispy_forms_tags %}
+{% block coupon_content %}
+<div class="py-4">
+    <form id="edit-coupon-form" action="{% url 'coupon_edit' coupon_id=coupon.id %}" method="post">
+        {% csrf_token %}
+        <div class="d-flex flex-row justify-content-between px-4">
+            <h4>Edit Coupon - {{ coupon.code }}</h4>
```

Comment:
I think don't need the word 'Edit' in this title - let's keep consistent with batch edit